tools/windows/install-help/cal/customactiondata.cpp

```diff
@@ -98,17 +98,17 @@ bool CustomActionData::DoesUserExist() const
 
 const std::wstring &CustomActionData::UnqualifiedUsername() const
 {
-    return _unqualifiedUsername;
+    return _user.Name;
 }
 
 const std::wstring &CustomActionData::Username() const
```

Comment:
This could be called `FullyQualifiedUsername` to make it less ambiguous. It's obvious here because it's next to `UnqualifiedUsername`, but out of context it might be less obvious.